Ports/cavestory/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='cavestory'
+version='2.6.5-1'
+files=(
+    "https://github.com/nxengine/nxengine-evo/archive/refs/tags/v${version}.zip v${version}.zip a3a1a802eda46f9922065a35624ba94e3823649a7e71012e479ed91fcdb50255"
+    'https://www.cavestory.org/downloads/cavestoryen.zip cavestoryen.zip aa87fa30bee9b4980640c7e104791354e0f1f6411ee0d45a70af70046aa0685f'
+)
+auth_type='sha256'
+depends=(
+    'libjpeg'
+    'libpng'
+    'SDL2'
+    'SDL2_image'
+    'SDL2_mixer'
+    'SDL2_ttf'
+)
+workdir="nxengine-evo-${version}"
+launcher_name='Cave Story'
+launcher_category='Games'
+launcher_command='/usr/local/bin/nxengine-evo'
+icon_file='platform/switch/icon.jpg'
+
+post_fetch () {
+  mkdir -p ${workdir}/build
+}
+
+build () {
+  cd ${workdir}/build
+  cmake -DCMAKE_BUILD_TYPE=Release .. 
```

Comment:
You don't need the post-fetch step or any of the `cd workdir` steps.

`run cmake -B build "${configopts[@]}"` as `configure()` should do the trick if you set `useconfigure` to true.

and `run make -C build` as `build()` should also do the trick. see other cmake ports for an example.

How does this get access to our CMakeToolchain file? As written it looks like this will build for the host.

the release build type and CMakeToolchain file should be included in `configopts`

Reply:
I'm not sure how, but it certainly gets access to it. I don't have CMake installed but looking at the source code in CMakeLists.txt, it does nothing if the CMAKE_TOOLCHAIN_FILE environment variable is set on the host.